package.json

```diff
@@ -19,13 +21,18 @@
   ],
   "author": "Parse",
   "license": "MIT",
+  "files": [
+    "src",
+    "README.md"
+  ],
   "devDependencies": {
-    "@semantic-release/changelog": "5.0.1",
-    "@semantic-release/commit-analyzer": "8.0.1",
-    "@semantic-release/git": "9.0.0",
-    "@semantic-release/github": "7.2.3",
-    "@semantic-release/npm": "7.1.3",
-    "@semantic-release/release-notes-generator": "9.0.3",
+    "@babel/eslint-parser": "7.21.8",
+    "@semantic-release/changelog": "6.0.3",
```

Comment:
I'm not sure I understand, is this a note for you, or me to try something? If it's for me, what are the steps I need to do to validate this?

Reply:
Same as https://github.com/parse-community/parse-server/pull/8643#issuecomment-1597294393

Reply:
From what I can tell, dry-runs seem to be for people who access to the original repo, I don't see much about parameters to point to my fork. Below is what I get:

```bash
[3:55:46 PM] [semantic-release] › ℹ  Running semantic-release version 21.0.1
Running on branch: main
Changelog file output to: ./CHANGELOG.md
[3:55:47 PM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/changelog"
[3:55:47 PM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/npm"
[3:55:47 PM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/git"
[3:55:47 PM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/github"
[3:55:47 PM] [semantic-release] › ✔  Loaded plugin "analyzeCommits" from "@semantic-release/commit-analyzer"
[3:55:47 PM] [semantic-release] › ✔  Loaded plugin "generateNotes" from "@semantic-release/release-notes-generator"
[3:55:47 PM] [semantic-release] › ✔  Loaded plugin "prepare" from "@semantic-release/changelog"
[3:55:47 PM] [semantic-release] › ✔  Loaded plugin "prepare" from "@semantic-release/npm"
[3:55:47 PM] [semantic-release] › ✔  Loaded plugin "prepare" from "@semantic-release/git"
[3:55:47 PM] [semantic-release] › ✔  Loaded plugin "publish" from "@semantic-release/npm"
[3:55:47 PM] [semantic-release] › ✔  Loaded plugin "publish" from "@semantic-release/github"
[3:55:47 PM] [semantic-release] › ✔  Loaded plugin "addChannel" from "@semantic-release/npm"
[3:55:47 PM] [semantic-release] › ✔  Loaded plugin "addChannel" from "@semantic-release/github"
[3:55:47 PM] [semantic-release] › ✔  Loaded plugin "success" from "@semantic-release/github"
[3:55:47 PM] [semantic-release] › ✔  Loaded plugin "fail" from "@semantic-release/github"
[3:55:49 PM] [semantic-release] › ⚠  Run automated release from branch main on repository https://github.com/parse-community/parse-server-fs-adapter in dry-run mode
[3:55:50 PM] [semantic-release] › ✘  The command "git push --dry-run --no-verify https://github.com/parse-community/parse-server-fs-adapter HEAD:main" failed with the error message remote: Permission to parse-community/parse-server-fs-adapter.git denied to cbaker6.
fatal: unable to access 'https://github.com/parse-community/parse-server-fs-adapter/': The requested URL returned error: 403.
[3:55:50 PM] [semantic-release] › ⚠  Skip step "fail" of plugin "@semantic-release/github" in dry-run mode
[3:55:50 PM] [semantic-release] › ✘  EGITNOPERMISSION Cannot push to the Git repository.
semantic-release cannot push the version tag to the branch main on the remote Git repository with URL https://github.com/parse-community/parse-server-fs-adapter.

This can be caused by:

    * a misconfiguration of the repositoryUrl (https://github.com/semantic-release/semantic-release/blob/master/docs/usage/configuration.md#repositoryurl) option
    * the repository being unavailable
    * or missing push permission for the user configured via the Git credentials on your CI environment (https://github.com/semantic-release/semantic-release/blob/master/docs/usage/ci-configuration.md#authentication)

SemanticReleaseError: Cannot push to the Git repository.
    at default (file:///Users/MacJr/Documents/github/parse-server-fs-adapter/node_modules/semantic-release/lib/get-error.js:6:10)
    at run (file:///Users/MacJr/Documents/github/parse-server-fs-adapter/node_modules/semantic-release/index.js:101:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Module.default (file:///Users/MacJr/Documents/github/parse-server-fs-adapter/node_modules/semantic-release/index.js:275:22)
    at async default (file:///Users/MacJr/Documents/github/parse-server-fs-adapter/node_modules/semantic-release/cli.js:55:5) {
  code: 'EGITNOPERMISSION',
  details: '**semantic-release** cannot push the version tag to the branch `main` on the remote Git repository with URL `https://github.com/parse-community/parse-server-fs-adapter`.\n' +
    '\n' +
    'This can be caused by:\n' +
    ' - a misconfiguration of the [repositoryUrl](https://github.com/semantic-release/semantic-release/blob/master/docs/usage/configuration.md#repositoryurl) option\n' +
    ' - the repository being unavailable\n' +
    ' - or missing push permission for the user configured via the [Git credentials on your CI environment](https://github.com/semantic-release/semantic-release/blob/master/docs/usage/ci-configuration.md#authentication)',
  semanticRelease: true
}%                                                    
```

Reply:
The log I posted shows some `dry-run` comments:

```bash
[3:55:49 PM] [semantic-release] › ⚠  Run automated release from branch main on repository https://github.com/parse-community/parse-server-fs-adapter in dry-run mode
[3:55:50 PM] [semantic-release] › ✘  The command "git push --dry-run --no-verify https://github.com/parse-community/parse-server-fs-adapter HEAD:main" failed with the error message remote: Permission to parse-community/parse-server-fs-adapter.git denied to cbaker6.
fatal: unable to access 'https://github.com/parse-community/parse-server-fs-adapter/': The requested URL returned error: 403.
[3:55:50 PM] [semantic-release] › ⚠  Skip step "fail" of plugin "@semantic-release/github" in dry-run mode
```

I also changed dry-run in `release.config.js`, but it did the same thing.

Reply:
I see, it tries a git push dry-run, hence the error. You could try to disable that if possible or just remove the git step temporarily from the config file, so it doesn't try to push a tag.

Reply:
I can't get this to work... It seems since you have access, it will be easier for you to attempt the dry run since you are familiar with the semantic-release package and I am not.

Reply:
I'll take a look